service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RootControllerTest.java

```diff
@@ -16,6 +21,15 @@
 @WebMvcTest
 public class RootControllerTest {
 
+    @MockBean
```

Comment:
Since these mocks are not required by the root endpoint, only required to start application I would rather create test application configuration with pointers to non existing locations e.g. 'http://fake-service-auth-provider-api.